exercises/concept/captains-log/.meta/src/reference/java/CaptainsLog.java

```diff
@@ -17,7 +17,7 @@ char randomPlanetClass() {
 
     String randomShipRegistryNumber() {
         var start = 1000;
-        var end = 9999;
+        var end = 11000;
```

Comment:
I think this should be `10000`, not `11000`: the numbers should be in the range `1000`-`9999`, so the upper limit should be `9999`+`1`.